build/common.project.props

```diff
@@ -12,7 +12,8 @@
     <NETFXTargetFramework>net472</NETFXTargetFramework>
     <NETCoreTargetFramework>netcoreapp2.1</NETCoreTargetFramework>
     <NETCoreTargetFrameworks Condition=" '$(RequiresSigningXplatAPIs)' != 'true' ">netcoreapp2.1</NETCoreTargetFrameworks>
-    <NETCoreTargetFrameworks Condition=" '$(RequiresSigningXplatAPIs)' == 'true' ">netcoreapp2.1;netcoreapp3.0</NETCoreTargetFrameworks>
+    <NETCoreTargetFrameworks Condition=" '$(RequiresSigningXplatAPIs)' == 'true' and '$(TestForOneSDK)' != 'true' ">netcoreapp2.1;netcoreapp3.0</NETCoreTargetFrameworks>
```

Comment:
As mentioned in the mac agent pool email thread, since we're only inserting into .net core 3, we don't need to test against .net core 2.x any longer. Can we remove `netcoreapp2.1` here?

Reply:
As discussed offline, ignore this comment.